tasks.py

```diff
@@ -0,0 +1,32 @@
+from invoke import run
+from invoke import task
+
+
+@task
+def clean(docs=False, bytecode=True, extra=''):
```

Comment:
Does it make sense to call `python setup.py clean` (or optionally `clean --all`) instead of building commands manually? It's likely less prone to user error.